tutorial/compiler.md

```diff
@@ -51,3 +51,17 @@ E.g.
 ```shell
 pyccel --compiler=PGI --language=c --export-compile-info=icc.json
 ```
+## Utilising Pyccel within a Conda Environment
```

Comment:
I think it is important to mention `Anaconda` here. Generally people using `conda` are more aware of versioning problems and know that the two are related but people that use `Anaconda` are not always aware of `conda` or of environments in general